### PR TITLE
use LOGGER.error to output Exception stack for Problem localization

### PR DESCRIPTION
**Pull request by 597365581**
use LOGGER.error to output Exception stack for Problem localization

**PR Checklist**

- [x] A description of the changes is added to the description of this PR.
- [ ] If there is a related issue, make sure it is linked to this PR.
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added or modified a feature, documentation in `docs` is updated

**Description of changes**

### CODE REVIEW

**Comment by 597365581** [comment]
@tdas 
use LOGGER.error to output Exception stack for Problem localization ，please help to review

File: server/src/main/java/io/unitycatalog/server/utils/JsonUtils.java [hunk 1]
```diff
@@ -48,8 +52,10 @@ public static void printJsonAsTable(String jsonString) {
             }
 
         } catch (JsonMappingException e) {
+            LOGGER.error("printJsonAsTable happen JsonMappingException,jsonString:{}",jsonString,e);
```
Comment (vikrantpuppala) [inline server/src/main/java/io/unitycatalog/server/utils/JsonUtils.java:55]: can we please format this better and add a meaningful message?

File: server/src/main/java/io/unitycatalog/server/utils/JsonUtils.java [hunk 2]
```diff
@@ -48,8 +52,10 @@ public static void printJsonAsTable(String jsonString) {
...
             System.out.println(jsonString);
```
Comment (vikrantpuppala) [inline server/src/main/java/io/unitycatalog/server/utils/JsonUtils.java:56]: Can we remove the system out if we add the log?

Comment (597365581) [reply server/src/main/java/io/unitycatalog/server/utils/JsonUtils.java:56]: I think it's no problem

File: server/src/main/java/io/unitycatalog/server/utils/JsonUtils.java (hunk 1 above)
Comment (597365581) [reply server/src/main/java/io/unitycatalog/server/utils/JsonUtils.java:55]: Ok，it's no problem，thanks

**Review by 597365581 — COMMENTED** [review]
I will modify

**Pull request closed without merging**